client/state/sharing/publicize/actions.js

```diff
@@ -110,36 +107,17 @@ export function fetchConnections( siteId ) {
  * @returns {Function}            Action thunk
  */
 export function fetchConnection( siteId, connectionId ) {
-	return ( dispatch ) => {
-		dispatch( {
-			type: PUBLICIZE_CONNECTION_REQUEST,
-			connectionId,
-			siteId,
-		} );
-
-		return wpcom.req
+	return ( dispatch ) =>
+		wpcom.req
 			.get( `/sites/${ siteId }/publicize-connections/${ connectionId }` )
 			.then( ( connection ) => {
 				dispatch( {
 					type: PUBLICIZE_CONNECTION_RECEIVE,
 					connection,
 					siteId,
 				} );
-				dispatch( {
-					type: PUBLICIZE_CONNECTION_REQUEST_SUCCESS,
-					connectionId,
-					siteId,
-				} );
 			} )
-			.catch( ( error ) =>
-				dispatch( {
-					type: PUBLICIZE_CONNECTION_REQUEST_FAILURE,
-					connectionId,
-					error,
-					siteId,
-				} )
-			);
-	};
+			.catch( () => {} );
```

Comment:
It's not part of this PR but I wonder what should actually happen if that request fails. I wasn't able to test this flow and there wasn't anything happening before so I guess this is just fine.

Reply:
Following these test instructions will help you test: #37517

I've tested that, and it worked well, so I think it's good to go.